html/modules/custom/reliefweb_entities/src/Services/ReportFormAlter.php

```diff
@@ -465,7 +465,6 @@ protected function alterFieldsForContributors(array &$form, FormStateInterface $
     // Hide fields.
     $form['field_bury']['#access'] = FALSE;
     $form['field_feature']['#access'] = FALSE;
-    $form['field_notify']['#access'] = FALSE;
```

Comment:
Not seeing the field as a contributor. I think the template needs to be updated as well: https://github.com/UN-OCHA/rwint9-site/blob/e8b6ca0bc2b7d8fd4d5615d2385a5e2d6186ee21/html/themes/custom/common_design_subtheme/templates/form/node-edit-form--report.html.twig#L124-L131